### ISSUE TITLE
Adding documentation for functions with multiple parameters?

### ISSUE THREAD

**Issue by Numpsy** [Zaid-Ajaj/Fable.Remoting#241]
Hi,

Say that I have this example from the documentation:

```fsharp
type IMusicStore = {
    createAlbum : string -> string -> DateTime -> Async<Option<Album>>
}
```

I tried plugging in documentation entries for that, and got:

![image](https://user-images.githubusercontent.com/1178570/116388299-96a25c80-a813-11eb-9f2b-0d302816b54d.png)

Should that work, or am I doing something silly?

Thanks

**Comment by Zaid-Ajaj** [comment]
Not sure why it won't compile 🤔 generally if you many arts, using a record with named fields will much better since you can what is what more clearly

**Comment by Numpsy** [comment]
Yes, I have that approach working seperately (with a pattern of TRequest -> Async<TResponse>), I just wasn't sure if this approach was supposed to work or not

Thanks

**Comment by Zaid-Ajaj** [comment]
Hi @Numpsy is this issue still valid? can you please give it another go with latest packages?

**Comment by Numpsy** [comment]
I'll give it a try later.

**Comment by Numpsy** [comment]
Hi,
It still looks the same to me using Fable.Remoting.AspNetCore 2.22.0 (this is using the AspNetCore bits directly, so no Giraffe etc, for the record.)

**Comment by vKito** [comment]
I'm also experiencing this issue. I also can't see any examples in the documentation UI.

**Comment by kerams** [comment]
Isn't the problem simply the fact that there's [support only for 1 and 2 parameter functions](https://github.com/Zaid-Ajaj/Fable.Remoting/blob/2d88a987d68f69ab4172c8563f3401a40dbe1e4d/Fable.Remoting.Server/Documentation.fs#L13-L38)? I guess it's not possible to generalize this with an expression taking functions with arbitrary arity?

**Comment by vKito** [comment]
Yup, I refactored my endpoint to accept a single value instead. Works this way thanks. At the same time I feel the need to point out a gotcha with the examples:

This is more a general quoted expression gotcha but I thought I would point it out for anyone needing help. If you pass in values that get computed at runtime, e.g. GUIDs, then the docs UI won't show any examples, but everything still compiles fine.
```
docs.route <@ fun api -> api.AddProductToCart @>
|> docs.example <@ fun api -> api.AddProductToCart(Guid.NewGuid(), Guid.NewGuid() @>  // example Guids for cart id and product id
```

The woraround to this is to simply create the GUID outside the quoted expression
```
let cartId = Guid.NewGuid()
let productId = Guid.NewGuid()
docs.example <@ fun api -> api.AddProductToCart(cartId, productId) @>
```
